Eshop.WebApi/Controllers/CategoryController.cs

```diff
@@ -7,7 +8,14 @@ namespace Eshop.WebApi.Controllers
     [Route("[controller]")]
     public class CategoryController : Controller
     {
-        public static List<Category> Categories = new List<Category>
+        private readonly EshopDbContext DbContext;
```

Comment:
private fields are camelCase syntax - the you willl need to do in ctor this.dbContext = dbContext because param is same like the field.
we have fields, constanst, properties, methods or parameters in methods and all are having some coding guidlines how all developers should write them

Reply:
fixed